src/lib.rs

```diff
@@ -70,15 +70,30 @@ where
 fn sanitize_path(uri: &mut Uri) {
     let path = uri.path();
     let path_decoded = decode(path);
+
+    // Check if the path contains a trailing slash and that it is not the only
+    // character.
```

Comment:
Maybe we should add a test for the root path (`/`) too?

Reply:
We do have this test: https://github.com/shuttle-hq/tower-sanitize-path/blob/main/src/lib.rs#L138-L144